ERCS/erc-7811.md

```diff
@@ -0,0 +1,222 @@
+---
+eip: 7811
+title: Wallet Asset Discovery
+description: JSON-RPC method for wallets to share a user’s full asset list with Dapps, including assets not easily found through on-chain data alone
+author: Luka Isailovic (@lukaisailovic), Konrad Kopp (@kopy-kat)
+discussions-to: https://ethereum-magicians.org/t/erc-7811-wallet-asset-discovery/21639
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-07
+---
+
+## Abstract
+
+This ERC introduces a new RPC call, wallet_getAssets, for wallets to declare to the Dapp what assets are owned by the user. This allows for more accurate asset discovery and the use of assets that aren’t available on-chain but can be provided by the wallet
```

Comment:
```suggestion
This ERC introduces a new RPC call, `wallet_getAssets`, for wallets to declare to the Dapp what assets are owned by the user. This allows for more accurate asset discovery and the use of assets that aren’t available on-chain but can be provided by the wallet
```

Please put code in backticks, both here and throughout the rest of the document.